webtestingii/src/App.test.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { render, fireEvent } from "react-testing-library";
+import "jest-dom/extend-expect";
+import App from "./App";
+
+describe("<App />", () => {
+  it("renders without crashing", () => {
+    const div = document.createElement("div");
+    ReactDOM.render(<App />, div);
+    ReactDOM.unmountComponentAtNode(div);
+  });
+
+  it("renders successfully", () => {
+    render(<App />);
+  });
+
+  it("successfully strikes", () => {
+    const { getAllByText, getByText } = render(<App />);
+    const button = getAllByText(/strike/i)[1];
+    fireEvent.click(button);
+    getByText(/1/);
+  });
+  it("successfully resets strikes", () => {
+    const { getAllByText, getByText } = render(<App />);
+    const button = getAllByText(/strike/i)[1];
+    fireEvent.click(button);
+    fireEvent.click(button);
+    fireEvent.click(button);
+    getByText(/0/);
```

Comment:
same thing as above, this will just pass if it finds _any_ text matching "0" in the DOM